packages/draft-js-export-html/src/stateToHTML.js

```diff
@@ -30,7 +30,7 @@ type RenderConfig = {
   style?: StyleDescr;
 };
 
-type BlockRenderer = (block: ContentBlock) => ?string;
+type BlockRenderer = (block: ContentBlock, generator: MarkupGenerator) => ?string;
```

Comment:
The second parameter should be optional.
```suggestion
type BlockRenderer = (block: ContentBlock, generator?: MarkupGenerator) => ?string;
```